pages/platform/kubernetes-k8s/installing-kubernetes-dashboard/guide.en-gb.md

```diff
@@ -64,6 +64,12 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-r
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-rc7/aio/deploy/recommended.yaml
 ```
 
+### For Kubernetes 1.20, choose the [latest version](https://github.com/kubernetes/dashboard/releases)
```

Comment:
Could indicate the release number instead of the "latest version" please?

Reply:
Done

Reply:
Thanks.